docs/Encyclopedia/pages/MuSE_Annotation.md

```diff
@@ -0,0 +1,29 @@
+# MuSE Annotation
+
+## Description ##
+
+MuSE Annotation is a somatic mutation annotation pipeline in GDC whole exome sequencing (WXS) and targeted sequencing harmonization. 
+
+## Overview ##
+
+MuSE Annotation is a workflow in the GDC that annotates somatic variants identified by the MuSE variant calling pipeline. Somatic mutation annotation is performed with MuSE Annotation using tumor and normal alignments and generates simple nucleotide variation (SNV) data.
+
+### Input
+
+* Tumor VCF - WXS or Targeted Sequencing
+* Normal VCF - WXS or Targeted Sequencing
```

Comment:
The tumor and normal BAM are compared to generate the VCF, which is effectively a comparison of the tumor and normal DNA.  I would recommend:

`* VCF (Data Type:  Raw Simple Somatic Mutation)`

Reply:
changed